build.gradle

```diff
@@ -1972,6 +1974,11 @@ project(':clients:clients-integration-tests') {
     testImplementation libs.slf4jApi
     testImplementation project(':test-common:test-common-internal-api')
     testImplementation project(':test-common:test-common-runtime')
+    testImplementation project(':metadata')
+    testImplementation project(':server')
+    testImplementation project(':storage')
+    testImplementation project(':core').sourceSets.test.output
+    testImplementation project(':clients').sourceSets.test.output
```

Comment:
It seems diplicate of `implementation project(':clients').sourceSets.test.output`.

Reply:
Updated it. Thanks.